sw/device/tests/BUILD

```diff
@@ -3570,6 +3570,14 @@ opentitan_test(
         EARLGREY_TEST_ENVS,
         EARLGREY_SILICON_OWNER_ROM_EXT_ENVS,
     ),
+    silicon = silicon_params(
+        test_cmd = """
+            --bootstrap="{firmware}"
+            --vbus-sense-en=VBUS_SENSE_EN
+            --vbus-sense=VBUS_SENSE
```

Comment:
Any reason to not make them the default in usb.rs?

It's a shame that we have a ton of code duplication here, but Bazel doesn't allow function definitions in BUILD...